modules/module_http_test.go

```diff
@@ -0,0 +1,32 @@
+//go:build linux
+// +build linux
```

Comment:
These build-constraints probably don't need to be here - the check is OS-agnostic.